<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>learning</class>
 <widget class="QDialog" name="learning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>175</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Обучение нейронной сети</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,0">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Прогресс выполнения</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	background-color: white;
	border: 1.5px solid black;
}

QProgressBar::chunk {
	background-color: rgb(255,192,203);
    width: 15px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
      <item>
       <widget class="QPushButton" name="pushButton_start_learn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Запустить</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_1">
          <property name="minimumSize">
           <size>
            <width>83</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>83</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Кол-во эпох</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimumSize">
           <size>
            <width>33</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>33</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop_learn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Остановить </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
